chapters/project-guide/project-guide.rst

```diff
@@ -109,7 +106,7 @@ Pods
 
 ``/pods``
 
-Our Flows will need some :term:`Pods<Pod>`, so we also create a dedicated folder for them. And you guessed right, the folder is called **pods**. Here we have 4 pods, but in your project, you might have a different number.
+Our Flows will need some :term:`Pods<Pod>`, so we also create a dedicated pods folder for them.
```

Comment:
Let's wrap pods in backticks. That way readers will know we're talking about a folder *called* `pods`, not just a folder for *containing* Pods